e2e/helper/validator_set.go

```diff
@@ -0,0 +1,47 @@
+package helper
```

Comment:
Seems it is better to put this to `e2e` package itself. Opinions?

Reply:
Makes sense, done